general/token/accesstokencreate.go

```diff
@@ -0,0 +1,150 @@
+package generic
```

Comment:
```suggestion
package login
```

Reply:
Not part of the login mechanism. Changed to `token`